docs/troubleshooting-oauth2.md

```diff
@@ -23,119 +23,116 @@ limitations under the License.
+#### Troubleshooting
 
+This issue is caused when the endpoint is not returning a JSON payload which matches with the [OpenId Connect Discovery Configuration](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig).
+These are the possible causes:
+- The payload returned by the endpoint is not compliant because it is empty or it is missing some critical information. To identify the root cause, open the browser's javascript console and search for one of these possible error messages:
+  - `Payload does not contain openid configuration` This error occurs when the payload is empty or it is not a JSON payload.
+  - `Missing authorization_endpoint` This error occurs when the JSON attribute `authorization_endpoint` is missing.
+  - `Missing token_endpoint` This error occurs when the JSON attribute `token_endpoint` is missing.
+  - `Missing jwks_uri` This error occurs when the JSON attribute `jwks_uri` is missing.
+- The url is wrong. Check out with your administrator of your Identity Provider to get the correct url to the OpenId Connect Discovery endpoint.
 
 ### Not authorized {#not-authorized-error}
 
-#### Steps to reproduce the issue
+#### Steps to reproduce
 
-Open the root url of the management UI in the browser, click on the buttoh "Click here to logon" and enter the credentials requested by your Identity Provider. You are redirected back to the management UI with the following error:
+Open the root url of the management UI in the browser, click on the button "Click here to logon" and enter the credentials requested by your Identity Provider. You are redirected back to the management UI with the following error:
 
 ```
 Not authorized
 ```
 
-#### Troubleshoot the issue
+#### Troubleshooting
 
 This issue occurs when the token does not have enough scopes or permissions to access the management UI. You need at least one of these scopes or the equivalent scope:
-- `rabbitmq.tag:administrator`
-- `rabbitmq.tag:management`
-- `rabbitmq.tag:monitoring`
-- `rabbitmq.tag:policymaker`
+
+* `rabbitmq.tag:administrator`.
+* `rabbitmq.tag:management`.
+* `rabbitmq.tag:monitoring`.
+* `rabbitmq.tag:policymaker`.
 
 Follow these steps to find out which scopes or permissions are carried in the token:
-1. Open your browwser's developer tool (e.g. in Chrome or Firefox, right-click on the page and click on *Inspect* menu option)
-2. Go to the tab *Application*
-3. Select the option *Storage* > *Local Storage* in the left panel
-4. Click on the tree option which matches the URL of the management UI
-5. Select the Key *rabbitmq.credentials* in the right panel
-6. Copy its value
-7. Go to the url https://jwt.io
-8. Paste the value into the text field *Encoded*
-9. Look at the payload's text field *Decoded*
+1. Open your browwser's developer tool (e.g. in Chrome or Firefox, right-click on the page and click on *Inspect* menu option).
```

Comment:
change e.g to "for example" and again I don't think there is need for brackets. I would start "For example..." as a new sentence